### ISSUE TITLE
VSSpaceCode Plugin Doesn't Work In VSCodium On Linux Mint

### ISSUE THREAD

**Issue by SamuelBanya** [VSpaceCode/VSpaceCode#360]
## Bug description

When installing on VSCodium on Linux Mint, the prompt to auto-configure happened only once.

I missed it as the user since it got superseded by another plugin I was installing at the time.

It's the basic "Do you want the plugin to configure you settings for you" or whatever screen that was.

I tried installing and reinstalling a ton of times, and it won't prompt me the same way again unfortunately.

This matters because pressing spacebar doesn't do anything

## Expected behavior

I expect the prompt to work each time I try to install the plugin

**Comment by SamuelBanya** [comment]
I couldn't write the entire settings.json or keyboard.json file because its too long for Github.

**Comment by SamuelBanya** [comment]
Nevermind, using 'Ctrl + Shift + p' in VS Codium, and then typing 'VSpaceCode: Configure Default Settings and Keybindings' command, and hitting enter fixed it, which I found by searching the issues page:
- https://github.com/VSpaceCode/VSpaceCode/issues/69

I saved the settings file it generated, and it seems like it's working just fine, thanks!

**Comment by SamuelBanya** [comment]
Note, this also works on MacOS as well.
